### DISCUSSION TITLE
Enregistement et envoie automatique de tableau

### DISCUSSION THREAD

**Discussion by simhelio** [witsa/synapps#574] [Q&A]
Bonjour

j'ai besoin d’enregistrer, tout les jours à 7h ,une liste de valeurs d'un tableau faisant partie d'une scène. 
Ces tableaux doivent être envoyer par mail ou via le réseau car un ordinateur est câblé en permanence. C'est le même principe qu'un bilan mais avec 200 lignes de données. Je ne vois pas comment faire. Faut il développer un composite? Merci de votre aide.

Ci-dessous un extrait des tableaux:

![image](https://user-images.githubusercontent.com/128216403/226094670-5ec86faf-e7af-4784-a076-e115c63fc168.png)

**Comment by WITNicolas** [comment]
Bonjour,

Il faut différencier 2 choses. 
STUDIO n'est entre guillement qu'une IHM d'exploitation afin de visualiser et agir sur des consignes.
Le fait de vouloir générer de la Data et d'envoyer ces dernieres par mail, relève du process du REDY en lui-même. Pour cela il faudra réaliser un script qui générera un fichier CSV qui pourra effectivement être envoyé par mail et ça ne sera "que" de la donnée brute.

La paire REDY / STUDIO ne remplace pas totalement un logiciel de SUPERVISION au sens propre du terme (comme EASY PILOT par exemple) qui, elle, bénéficie de la puissance d'une base de donnée type SQL et de tous les outils pouvant l'exploiter comme REPORTING SERVICES. Ce dernier permet de générer des raports / camembert / courbe / tableau / histogramme en format PDF (entre autre)
